### DISCUSSION TITLE
Top language card is empty

### DISCUSSION THREAD

**Discussion by sudhdkso** [anuraghazra/github-readme-stats#2301] [Q&A] [answered]
### Describe the bug

hi! 
I'm using the github status you made well.
I'm trying to add a language card this time. But my language card doesn't show anything.
So I looked for problems similar to my situation.
After that, I read the FAQ and sent the query.
I don't think there's a problem with the results.
But it still doesn't work.

### Expected behaviour

Language should be displayed well.

### Screenshots / Live demo link

<img width="318" alt="image" src="https://user-images.githubusercontent.com/78727847/204291433-9161cf37-d3fc-4e92-aa88-359a9bd976c2.png">

**Comment by bylickilabs** [comment, marked as answer]
@sudhdkso 
edit your README and write by your own username: username=sudhdkso .......

**Comment by bylickilabs** [comment]
@sudhdkso 

I tested the code with your username! It works

**Comment by rickstaa** [comment]
@bylickilabs is correct. You can use the following syntax:

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=sudhdkso)](https://github.com/anuraghazra/github-readme-stats)

```graphql
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=sudhdkso)](https://github.com/anuraghazra/github-readme-stats)```

**Reply by bylickilabs** [reply]
Thanks

**Comment by bylickilabs** [comment]
@rickstaa 
You are doing a great job, thank you for that! Use lots of your gadgets

**Comment by rickstaa** [comment]
> @rickstaa You are doing a great job, thank you for that! Use lots of your gadgets

Ah thanks! 😄 And thank you sir for helping us maintain this repo 🚀.

**Reply by bylickilabs** [reply]
You are doing a great job, thank you for that! Use lots of your gadgets

**Comment by bylickilabs** [comment]
@rickstaa 
It is my real joy and pleasure.
A project like this simply has to be supported with all available means

**Comment by bylickilabs** [comment]
@rickstaa 
Can you please help me to get the achievement Galaxy Brain. For this, two questions in which I was tagged must be answered in a discussion.

**Reply by rickstaa** [reply]
Done! 👍🏻.

**Reply by bylickilabs** [reply]
Thank you

**Reply by bylickilabs** [reply]
@rickstaa 
What should I do?

**Reply by rickstaa** [reply]
Nothing I already accepted your answer as the answer. After enough accepted answers, the badge shows up.

**Comment by bylickilabs** [comment]
@sudhdkso Did it work?

**Reply by rickstaa** [reply]
You can find more information about the github achievements at https://github.com/Schweinepriester/github-profile-achievements. For the first Galaxy brain, you need 2 accepted answers. Please be aware that a bug prevents the badge from showing up if you have accepted your own answer somewhere. I had to remove these for my badge to show up. 👍🏻

**Reply by sudhdkso** [reply]
I checked now and it works fine!
thank you 😊
